content/en/real_user_monitoring/reactnative/_index.md

```diff
@@ -36,13 +36,30 @@ To install with Yarn, run:
 yarn add @datadog/mobile-react-native
 ```
 
+### iOS
+
 Install the added pod:
 
 ```sh
 (cd ios && pod install)
 ```
 
-Versions `1.0.0-rc5` and higher require you to have `compileSdkVersion = 31` in the Android application setup, which implies that you should use Build Tools version 31, Android Gradle Plugin version 7, and Gradle version 7 or higher. To modify the versions, change the values in the `buildscript.ext` block of your application's top-level `build.gradle` file. Datadog recommends using React Native version 0.67 or higher.
+### Android 
+
+If you use a React Native version strictly over 0.67, make sure to use Java version 17. If you use React Native version equal or below ot 0.67, make sure to use Java version 11.
+
+In your `android/build.gradle` file, specify the `kotlinVersion` to avoid clashes among kotlin dependencies:
+
+```groovy
+buildscript {
+    ext {
+        // targetSdkVersion = ...
+        kotlinVersion = "1.8.21"
+    }
+}
+```
+
+The Datadog React Native SDK requires you to have `compileSdkVersion = 31` in the Android application setup, which implies that you should use Build Tools version 31, Android Gradle Plugin version 7, and Gradle version 7 or higher. To modify the versions, change the values in the `buildscript.ext` block of your application's top-level `build.gradle` file. Datadog recommends using a React Native version that's actively supported.
```

Comment:
maybe we can say `or higher` here for `compileSdkVersion` and build tools, given that the current one is 34?